### PR TITLE
better list format for example team GH orgs

### PR DESCRIPTION
**Pull request by stefaniebutland**
Also, lots of html files from _site

🤔 To created unordered list, I needed to add hard return before the first list item. This is not usually required and is not evident in [Quarto docs for Lists](https://quarto.org/docs/authoring/markdown-basics.html#lists)

### CODE REVIEW

**Comment by stefaniebutland** [comment]
@jules32 here's the error msg for failing check. I did not run into this last time. Can you please look into it?
My local render looked great.

```
Error: Can't find 'action.yml', 'action.yaml' or 'Dockerfile' under '/home/runner/work/_actions/quarto-dev/quarto-actions/main/install-quarto'. Did you forget to run actions/checkout before running your local action?
```

File: core-lessons/github/index.qmd [hunk 1]
```diff
@@ -45,24 +45,28 @@ Here are two examples of GitHub Organizations created as part of Openscapes Cham
 
 Many Champions teams create GitHub Organizations for their research group. This is a way for all the work that happens in the research group to be organized in one place, but also clearly attributed and credited by each user who contributes. 
 
-You can explore other Champions teams' GitHub organizations they've created:
+You can explore other Champions teams' GitHub organizations they've created. They're listed under the Cohorts in which they particpated:
```
Comment (jules32) [inline core-lessons/github/index.qmd:48]: typo in "participated"

**Review by jules32 — APPROVED** [review]
One small edit and then ok to merge!

File: core-lessons/github/index.qmd (hunk 1 above)
Comment (stefaniebutland) [reply core-lessons/github/index.qmd]: fixed thanks

**Comment by stefaniebutland** [comment]
d'oh! [failing check](https://github.com/Openscapes/series/pull/8#issuecomment-1164767604) noted in my comment means merging did not make the update. Can you please check this @jules32 ?

**Pull request merged**